docs/manual/common/guide/logging/code/log4j2-lagom-default.xml

```diff
@@ -12,7 +12,7 @@
 
     <Console name="STDOUT">
       <PatternLayout>
-        <Pattern>%d{yyyy-MM-dd'T'HH:mm:ss.SSS'Z'}{GMT+0} %highlight{%level} %logger{0} [%mdc] - %msg%n</Pattern>
+        <Pattern>%d{yyyy-MM-dd'T'HH:mm:ss.SSS'Z'}{GMT+0} %highlight{%level} %logger{10} [%mdc] - %msg%n</Pattern>
```

Comment:
OK I've checked the Logback and log4j docs, and found that though they both support an argument to `%logger`, they interpret it very differently, so it probably doesn't make sense to specify the same value for both.

| Library | Syntax | Logger name | Result |
| ---        | ---       | ---                  | ---      |
| Logback | `%logger{10}` | `mainPackage.sub.sample.Bar` | `m.s.s.Bar`                            |
| log4j       | `%logger{10}` | `org.apache.commons.Foo`       | `org.apache.commons.Foo` |






- https://logback.qos.ch/manual/layouts.html#ClassicPatternLayout
- https://logging.apache.org/log4j/2.x/manual/layouts.html#PatternLayout